app/src/main/java/com/google/maps/android/ktx/demo/io/MyItemReader.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.maps.android.ktx.demo.io
+
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.ktx.demo.model.MyItem
+import org.json.JSONArray
+import org.json.JSONException
+import java.io.InputStream
+import java.util.*
+
+/**
+ * Helper class to read in cluster items from a resource
+ */
+class MyItemReader {
+
+    /**
+     * Returns a list of cluster items read from the provided [inputStream]
+     */
+    @Throws(JSONException::class)
+    fun read(inputStream: InputStream): List<MyItem> {
+        // This matches only once in whole input so Scanner.next returns whole InputStream as a
+        // String. http://stackoverflow.com/a/5445161/2183804
+        val REGEX_INPUT_BOUNDARY_BEGINNING = "\\A"
+
+        val items: MutableList<MyItem> = ArrayList()
```

Comment:
You can use `mutableListOf()` here
```suggestion
        val items = mutableListOf<MyItem>()
```